gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -334,6 +334,11 @@
             <property name="message" value="printStackTrace is not generally allowed."/>
             <property name="ignoreComments" value="true"/>
         </module>
+        <module name="RegexpSinglelineJava">
```

Comment:
Please give this an `id` so that it can be suppressed using an `@SuppressWarnings` annotation, e.g.

```
<property name="id" value="BanSystemOut"/>
```

Reply:
Nice, I didn't know how to do that.

Reply:
Ok done, have a look